README.md

```diff
@@ -39,7 +39,13 @@ npm install opent2t-translator-com-wink-thermostat
 ```
 2. First step is to do the onboarding. Wink communicates via the hub so you need to set that up first
 ```bash
-node index.js -o opent2t-translator-com-wink-hub -h WinkHub
+node_modules\.bin\opent2t-cli -o opent2t-translator-com-wink-hub -h WinkHub
+```
+
+Or, if installed globally
```

Comment:
dont you need to install the translators globally too? or can you install them in a folder and then run from that folder?